OnBoardingCalculator/app/src/main/java/com/globant/onboardingcalculator/mvp/model/CalculatorModel.java

```diff
@@ -1,45 +1,57 @@
 package com.globant.onboardingcalculator.mvp.model;
 
-import com.globant.onboardingcalculator.utils.Constants;
+import java.text.DecimalFormat;
+
+import static com.globant.onboardingcalculator.utils.Constants.EMPTY_CHAR;
+import static com.globant.onboardingcalculator.utils.Constants.EMPTY_STRING;
+import static com.globant.onboardingcalculator.utils.Constants.MATH_ERROR;
+import static com.globant.onboardingcalculator.utils.Constants.NUMBER_ZERO;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_DIVIDE;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_MULTIPLY;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_PLUS;
+import static com.globant.onboardingcalculator.utils.Constants.OPERATOR_SUBSTRACTION;
+import static java.lang.Double.parseDouble;
 
 public class CalculatorModel {
     private String firstOperand;
     private String secondOperand;
     private char operator;
 
+    private DecimalFormat df = new DecimalFormat("#.##");
```

Comment:
pls put a name more representative, df if not representive

Reply:
Done! Thanks Bra!